client/src/app/Home/home.component.ts

```diff
@@ -1,9 +1,46 @@
-import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { Component, OnInit } from '@angular/core';
 
 @Component({
   selector: 'home-app',
   templateUrl: 'app/home/home.component.html',
+  styleUrls: ['app/home/home.component.css']
```

Comment:
that's something I haven't seen before, however I use webpack which doesn't work with the module id.